client/src/ui/modules/onboarding/StepsDevMode.tsx

```
@@ -0,0 +1,388 @@
+    } else {
+      setCopyMessage("No account to export");
+    }
+  };
+
+  const onImportAccount = () => {
+    navigator.clipboard.readText().then((text) => {
+      try {
+        const burner = JSON.parse(text);
+        const currentBurners = localStorage.getItem("burners") ? JSON.parse(localStorage.getItem("burners") || "") : {};
+
+        if (currentBurners.hasOwnProperty(burner.address)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)